.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: Found a bug? Report it with this template.
+
+body:
+- type: markdown
+  attributes:
+    value: |-
+      Thank you for opening a Bug report!
+      Please note that the issue tracker is reserved for issues, errors and feature requests.
+      
+      **Want to make a Feature request instead?**
+      Use the [Feature request](https://github.com/Chew/JDA-Chewtils/issues/new?template=feature_request.yml) template!
+      
+      - For examples, check the [examples module](https://github.com/Chew/JDA-Chewtils/tree/master/examples).
+      - For questions join the [Discord Server of Chew](https://discord.gg/SUGVxJpg8r) and ask in the `#jda-chewtils` channel.  
+        **Do not contact the JDA-Team for questions regarding JDA-Chewtils!**
+      - For general programming questions ask on [StackOwnerflow](https://stackoverflow.com/questions/tagged/java) instead.
+- type: checkboxes
+  attributes:
+    label: Issue Checklist
+    description: |-
+      Please follow the following steps before opening this issue.
+      Issues that do not complete the checklist may be closed without any help.
+    options:
+    - label: I have checked for similar issues on the [Issue tracker](https://github.com/Chew/JDA-Chewtils/issues)
+      required: true
+    - label: I have updated to the [latest version](https://github.com/Chew/JDA-Chewtils/releases) of JDA-Chewtils.
+      required: true
+    - label: I have checked the branches or the maintainer's PRs for upcoming bug fixes.
+      required: true
+- type: dropdown
+  attributes:
+    label: Affected Modules
+    description: Which modules do you have issues with? Multiple selections possible.
+    multiple: true
+    options:
+    - Command
+    - Commons
+    - Menu
+    - OAuth2
```

Comment:
Should examples or doc be added here?

Reply:
I think I can add the Examples one, but docs isn't really a module, no? More the folder of the generated Javadocs, right?